.cirrus.yml

```diff
@@ -935,6 +935,8 @@ minikube_test_task:
     alias: minikube_test
     # Docs: ./contrib/cirrus/CIModes.md
     only_if: *not_tag_magic
+    # 2024-05-21: flaking almost constantly since March.
+    skip: $CI == $CI
```

Comment:
Any reason to not just nuke the entire section?

Reply:
My foolish hope that by making it visible in .cirrus.yml and very easy to revert, someone -- probably me -- will be tempted to take a deep look at the problem one day? Emphasis on "hope" but, yeah, "foolish" is not too far behind.

Reply:
understood, I am fine to have it like this for now but if nobody is ever going to work on this again then removing is better
but yeah let's get CI in a better state for now

Reply:
I keep thinking it's something simple. The problem is consistently of the form:
```
$ minikube start
...
Get "https://localhost:43781/apis/apps/v1/namespaces/kube-system/deployments/coredns/scale": dial tcp [::1]:43781: connect: connection refused
```
so I would expect it to be easy to find who/where is trying to come up on that port. And every time I look, I get nowhere. I'm just trying to give myself (and everyone) breathing room.